blueman/main/indicators/AppIndicator.py

```diff
@@ -0,0 +1,33 @@
+# coding=utf-8
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import gi
+
+gi.require_version('AppIndicator3', '0.1')
```

Comment:
This will cause an exception if the typelib is not installed, if you are expecting this ignore this. If not see the current Appindicator plugin how I solved it.